CODEOWNERS

```diff
@@ -6,20 +6,11 @@
 ## CI config
 /.circleci/ @r-marques @tanertopal
```

Comment:
We have to remove this as well, it's obsolete

Reply:
Thanks for pointing it out. This was done in another PR #195.